src/com/tkjngine/gametable/monsterwounds/WoundsDialogue.java

```diff
@@ -0,0 +1,111 @@
+package com.tkjngine.gametable.monsterwounds;
+
+import java.awt.*;
+import java.awt.event.ActionListener;
+
+import javax.swing.*;
+
+public class WoundsDialogue extends JDialog {
```

Comment:
I think we could probably write a more generic input dialogue to simplify some of these.